### PR TITLE
Modify git clone to perform shallow clone

### PR DESCRIPTION
**Pull request by Shubhamr837**
What is included in this PR?
There was an error in step numbering that has been corrected.
git clone command for sync-endpoint is modified to perform shallow clone.

### CODE REVIEW

**Comment by Shubhamr837** [comment]
Hi @linl33 ! I modified the sync-endpoint-mannual-setup doc. But I am confused about the changes to be made in the yml files at https://github.com/odk-x/docs/tree/master/odkx-src/files. The cloud-init files have command clone the sync-endpoint-default-setup repository ,should we modify that? Please guide me about what changes are to be made.

**Comment by wbrunette** [comment]
@Shubhamr837 you are looking at the docker image setup files, the git clone happens when the user runs the install after the image is created because user input is needed. Here is a link
https://github.com/odk-x/sync-endpoint-default-setup/blob/master/init-odkx-sync-endpoint.py#L153

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 1]
```diff
@@ -52,62 +52,66 @@ Setup instructions:
...
   .. code-block:: console
 
-    $ git clone https://github.com/odk-x/sync-endpoint
-   
-  3. Navigate into the sync-endpoint directory. Most likely
+    $ git clone -b master --single-branch --depth=1 https://github.com/odk-x/sync-endpoint
+
+  .. note::
+
+     The above command performs a shallow clone of the sync-endpoint Github repository. To perform a full clone remove :guilabel:`--single-branch --depth=1` from the command.
```
Comment (linl33) [inline odkx-src/sync-endpoint-manual-setup.rst:59]: It is a git repository, not a GitHub repository. Or you could specify that it's a GitHub git repository.

**Review by linl33 — CHANGES_REQUESTED** [review]
I thought the cloud-init scripts also clone sync-endpoint. You don't need to make changes to the cloud-init scripts.

**Comment by Shubhamr837** [comment]
Thanks @linl33 I have made the changes , please review

File: odkx-src/sync-endpoint-manual-setup.rst [hunk 2]
```diff
@@ -52,62 +52,66 @@ Setup instructions:
...
   .. code-block:: console
 
-    $ git clone https://github.com/odk-x/sync-endpoint
-   
-  3. Navigate into the sync-endpoint directory. Most likely
+    $ git clone -b master --single-branch --depth=1 https://github.com/odk-x/sync-endpoint
+
+  .. note::
+
+     The above command performs a shallow clone of the sync-endpoint Github git repository. To perform a full clone remove :guilabel:`--single-branch --depth=1` from the command.
```
Comment (linl33) [inline odkx-src/sync-endpoint-manual-setup.rst:59]: > sync-endpoint Github git repository

It should be capitalized as GitHub

**Comment by Shubhamr837** [comment]
Thanks @linl33 I have made the changes. Please review and suggest if any other change is needed.

**Review by wbrunette — APPROVED** [review]

**Pull request merged**